app/code/Magento/Tax/view/adminhtml/layout/tax_rule_edit.xml

```diff
@@ -12,7 +12,7 @@
     </head>
     <body>
         <referenceContainer name="content">
-            <block class="Magento\Tax\Block\Adminhtml\Rule\Edit"/>
+            <block class="Magento\Tax\Block\Adminhtml\Rule\Edit" name="content.tax_rule_edit"/>
             <block class="Magento\Tax\Block\Adminhtml\Rule\Edit\Form" name="tax-rule-edit" template="Magento_Tax::rule/edit.phtml"/>
```

Comment:
Not sure. As for me - this is not BC change because you hadn’t ability to customize these parts, now you will have. I don’t know any case what these changes could brake existing website

Reply:
Note the line I commented on - after change names are
`name="content.tax_rule_edit"` and `name="tax-rule-edit"`

normally they should have been
`name="tax-rule-edit"` and `name="tax-rule-edit-form"`

Reply:
@orlangur, @ihor-sviziev Changing a name of a block is A BiC so let's keep `tax-rule-edit` even if it not consistent with other names.